### PR TITLE
Add Second Workshop Material

### PR DESCRIPTION
**Pull request by brdywltn**
I've added in the first notebook for session 2 that Anna and I created. Please shout if there are any issues.

### CODE REVIEW

File: Session2_Intro_To_Machine_Learning_For_Meteorological_Science/requirements.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - jupyterlab>=3.4.7
+  - pandas>=1.5.0
+  - numpy>=1.23.3
+  - matplotlib>=3.6.0
+  - scipy>=1.9.1
+  - ipykernel>=6.16.0
+  - ipython>=8.5.0
+  - tensorflow>=2.0
+  - scipy>=1.9.1
+  - sklearn>=1.0.2
```
Comment (HarveySouth) [inline Session2_Intro_To_Machine_Learning_For_Meteorological_Science/requirements.yml:15]: The package name for sklearn on conda is 'scikit-learn', the environment will resolve when the name is updated

File: Session2_Intro_To_Machine_Learning_For_Meteorological_Science/01_NASA_Space_Apps_Bootcamp_MLP_with_Jena_data.ipynb [hunk 2]
```diff
@@ -0,0 +1,2231 @@
+{
```
Comment (HarveySouth) [inline Session2_Intro_To_Machine_Learning_For_Meteorological_Science/01_NASA_Space_Apps_Bootcamp_MLP_with_Jena_data.ipynb:1]: <p>The title markdown will render with a space between the '#'s and the title text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/informatics-lab/spaceapps-2022-mo-bootcamp/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (HarveySouth) [inline Session2_Intro_To_Machine_Learning_For_Meteorological_Science/01_NASA_Space_Apps_Bootcamp_MLP_with_Jena_data.ipynb:1]: <p>If you would like, you can probably remove the CoLab disclaimer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/informatics-lab/spaceapps-2022-mo-bootcamp/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Comment (brdywltn) [reply Session2_Intro_To_Machine_Learning_For_Meteorological_Science/01_NASA_Space_Apps_Bootcamp_MLP_with_Jena_data.ipynb:1]: I've left this in for now until I speak to Anna later and work out what I'll replace it with

**Pull request merged**